File: 백준/Bronze/2566. 최댓값/최댓값.py
arr = []
locate = []

for i in range(9):
    arr.append(list(map(int, input().split()[:9])))

max = float('-inf')
for i in range(9):
    for j in range(9):
        if(max < arr[i][j]):
            max = arr[i][j]
            locate = [[i, j]]
        elif(max == arr[i][j]):
            locate.append([i, j])
print(max)
print('{} {}'.format(locate[0][0] + 1, locate[0][1] + 1))

File: 백준/Bronze/2566. 최댓값/최댓값.c
#include <stdio.h>
#include <stdlib.h>


int main(void) {
	int arr[9][9];
	int max = 0, x = 1, y = 1;
	for (int i = 0; i < 9; i++) {
		for (int j = 0; j < 9; j++) {
			scanf("%d", &arr[i][j]);
			if (max < arr[i][j]) {
				max = arr[i][j];
				x = i + 1;
				y = j + 1;
			}
		}
	}
	printf("%d\n%d %d", max, x, y);
	
	return 0; 
}